docs/operators/regexparser.md

```diff
@@ -0,0 +1,266 @@
+User needs to set the `schema` JSON string which describes data as well as specify the format on date fields if any.
+e.g.
+
+```xml
+    <property>
+        <name>dt.application.{ApplicationName}.operator.{OperatorName}.prop.schema</name>
+        <value>{
+                 "fields": [
+                   {
+                     "name": "date",
+                     "type": "Date",
+                     "constraints": {
+                       "format": "yyyy:MM:dd:hh:mm:ss"
+                     }
+                   },
+                   {
+                     "name": "id",
+                     "type": "Integer"
+                   },
+                   {
+                     "name": "signInId",
+                     "type": "String"
+                   },
+                   {
+                     "name": "ipAddress",
+                     "type": "String"
+                   },
+                   {
+                     "name": "serviceId",
+                     "type": "Double"
+                   },
+                   {
+                     "name": "accountId",
+                     "type": "Long"
+                   },
+                   {
+                     "name": "platform",
+                     "type": "Boolean"
+                   }
+                 ]
+               }
+        </value>
+    </property>
+```
+
+User needs to set the `splitRegexPattern` property whose value is the regular expression that describes the pattern of the incoming data.
+
+Example for setting `splitRegexPattern` from properties.xml of the application.
+
+```xml
+    <property>
+        <name>dt.application.{ApplicationName}.operator.{OperatorName}.prop.splitRegexPattern</name>
+        <value>.+\[SEQ=\w+\]\s*(\d+:[\d\d:]+)\s(\d+)\s* sign-in_id=(\S+) .*ip_address=(\S+).* service_id=(\S+).*account_id=(\S+).*platform=(\S+)</value>
+    </property>
+```
+
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *schema* | [Schema](https://github.com/apache/apex-malhar/blob/master/contrib/src/main/java/com/datatorrent/contrib/parser/DelimitedSchema.java)  describing data | String | YES | N/A |
+| *splitRegexPattern* | regex expression that describes the pattern of incoming data | String | YES | N/A |
+
+ 
+## Platform Attributes that influence operator behavior
+
+| **Attribute** | **Description** | **Type** | **Mandatory** |
+| -------- | ----------- | ---- | ------------------ |
+| *TUPLE_CLASS* | TUPLE_CLASS attribute on output port which tells operator the class of POJO which need to be emitted | Class| Yes |
+ 
+The operator takes care of converting the byte array received on the input port to a string by decoding using the JVM's default charset. Then, splits the string using the `splitRegexPattern` and populates an object using the `schema`. Apex platform converts this object to the object of `TUPLE_CLASS` attribute value while emitting.
```

Comment:
This needs more explanation --  the connection between the regular expression and the fields of the POJO is not clear.